src/schema/mod.rs

```diff
@@ -153,6 +153,8 @@ pub use self::term::{Term, ValueBytes, JSON_END_OF_PATH};
 pub use self::text_options::{TextFieldIndexing, TextOptions, STRING, TEXT};
 pub use self::value::Value;
 
+pub(crate) const DEFAULT_FAST_FIELD_TOKENIZER: &str = "default";
```

Comment:
nitpick: don't we want to call it by its real name `lower`?

Reply:
I think `TokenizerManager::default_for_fast_fields` registers it both under `lower` and under `default`. If I understand things correctly, using `default` would mean the concrete choice is subject to change whereas `lower` should always yield the lower case even in future Tantivy versions?

Reply:
I would prefer `default_fast_field`, to avoid confusing it with `default`

Reply:
@adamreichold "default" is the one that will be used when a user is not specifying any tokenizer.
For isntance, when using the flags.

`.add_text(my_field, STRING | FAST)`

It is then possible in tantivy to pass your own tokenizer manager for indexing and for fast fields.
So not putting the specific tokenizer directly here makes it possible for users to pass a tokenizer manager with the default tokenizer they prefer.